### PR TITLE
Containerize Jetski

### LINKED ISSUE #12: Containerize JetSki
While we use containers at various places in the code like running badfish commands/hammercli commands, we should containerize JetSki so that users have a consistent deployment experience regardless of what OS they are running JetSki from.

### PR DESCRIPTION
**Pull request by jaredoconnell**
# Description

These changes make it easy to run JetSki in a container.

The image is created from a CentOS 8. It incorporates Ansible, its dependencies, packages needed to run some of the things in the container, hammercli, badfish, and a copy of the master branch of redhat-performance's JetSki.

The container sets an environment variable "DISABLE_PODMAN" to true. That environment variable is what it listens to for checking to see if it should create containers for hammercli and badfish. This prevents the container in a container complication. If there is a way that you think could be better, let me know.
The convenience script that runs the container passes in the two main configuration files the use may change into the container.

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes #12 

## Type of change
- New feature (non-breaking change which adds functionality)

## Testing

Here is the verbose output of the early log output: http://pastebin.test.redhat.com/880594 
It appeared to run correctly.

**Test Configuration**:
Client:
- Software: Fedora 32 with podman installed.

Server:
- Cloud15
- Hardware: fc640

## Checklist

- [x] I have performed a self-review of my own code
- [x] If a change is adding a feature, it should require a change to the README.md and the review should catch this.
- [x] If the change is a fix, it should have an issue. The review should make sure the comments state the issue (not just the number) and it should use the keywords that will close the issue on merge.
- [ ] A change should not being merged unless it passes CI or there is a comment/update saying what testing was passed.
- [ ] PRs should not be merged unless positively reviewed.

### CODE REVIEW

File: ansible-ipi-install/roles/bootstrap/tasks/20_reprovision_nodes.yml [hunk 1]
```diff
@@ -22,40 +22,73 @@
   set_fact:
     os_install: "{{ (rhel_version is version('8.1', '<') or rebuild_provisioner) | ternary('RHEL 8.1', false) }}"
 
+- name: Determine if containers should be used
+  set_fact:
+    run_in_container: "{{ lookup('env', 'DISABLE_PODMAN') != 'true' }}"
```
Comment (smalleni) [inline ansible-ipi-install/roles/bootstrap/tasks/20_reprovision_nodes.yml:27]: The reference to `DISABLE_PODMAN`  here seems to be counter-intuitive. I understand that this disables running hammercli etc inside a separate container, but this also makes it look like we are not running containerized jetski in a container. Can we come up with something else?

File: ansible-ipi-install/group_vars/all.yml [hunk 2]
```diff
@@ -9,7 +9,7 @@ ansible_ssh_key: "{{ ansible_user_dir }}/.ssh/id_rsa"
 # The version of the openshift-installer, undefined or empty results in the playbook failing with error message.
 # Values accepted: 'latest-4.3', 'latest-4.4', explicit version i.e. 4.3.0-0.nightly-2019-12-09-035405
 # For reference, https://openshift-release.svc.ci.openshift.org/
-version: "4.4.4"
+version: "latest"
```
Comment (smalleni) [inline ansible-ipi-install/group_vars/all.yml:12]: Let's not change this....

**Comment by smalleni** [comment]
Also leaving a commit when we get to merging this patch, we need to `Squash and merge`.

File: Dockerfile [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+ENV ansible_dir=/root/JetSki/ansible-ipi-install
+ENV DISABLE_PODMAN=true
+
+RUN yum -y install python3 --nodocs
+RUN pip3 install ansible
+RUN yum -y install epel-release --nodocs
+RUN yum --enablerepo=epel -y install sshpass
+RUN yum -y install openssh-clients --nodocs
+RUN yum -y install git --nodocs
+RUN pip3 install jmespath
```
Comment (smalleni) [inline Dockerfile:12]: What do we need this for?

File: Dockerfile [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+RUN pip3 install j2cli
+
+# Done with hammercli. Next badfish
+# Source: https://github.com/redhat-performance/badfish/blob/master/Dockerfile
+
+RUN git clone https://github.com/redhat-performance/badfish /root/badfish
+
+WORKDIR /root/badfish
+RUN pip3 install -r requirements.txt
+RUN python3 setup.py build
```
Comment (smalleni) [inline Dockerfile:32]: Are the build and install steps needed? I reckon just installing the requirements should work and that's what we do in the `shared-labs` role too.

File: Dockerfile (hunk 3 above)
Comment (jaredoconnell) [reply Dockerfile:12]: Several repositories here use epel.

openssh-clients is needed for at least one task run by the playbook. It cannot connect to a remote server over SSH without it.
Git is needed to clone the repository.
jmespath is another requirement of at least one task run by the playbook.

Comment (smalleni) [reply Dockerfile:12]: Yeh I was curious about jmespath not the others..cool

File: Dockerfile (hunk 4 above)
Comment (jaredoconnell) [reply Dockerfile]: You appear to be correct. I tested it without it and it still worked. The commit has been pushed.

File: Dockerfile [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+
+# Done with hammercli. Next badfish
+# Source: https://github.com/redhat-performance/badfish/blob/master/Dockerfile
+
+RUN git clone https://github.com/redhat-performance/badfish /root/badfish
+
+WORKDIR /root/badfish
+RUN pip3 install -r requirements.txt
+
+ADD https://api.github.com/repos/redhat-performance/JetSki/git/refs/heads/containerized version.json
```
Comment (smalleni) [inline Dockerfile:33]: @jaredoconnell why do we need this?

Comment (smalleni) [reply Dockerfile:33]: Also shouldn't this be  https://api.github.com/repos/redhat-performance/JetSki/git/refs/heads/master and this is being downloaded into /root/badfish which doesn't sound right.

Comment (jaredoconnell) [reply Dockerfile:33]: Good catch. I will fix that.

Comment (jaredoconnell) [reply Dockerfile]: Fixed. I used an absolute path. I also fixed the branch.

**Pull request merged**